package.json

```diff
@@ -15,10 +15,14 @@
     "clean": "nx run-many -t clean",
     "version:update": "nx run-many -t version:update",
     "compile": "nx run-many -t compile",
+    "compile:ci:affected": "nx affected -t compile --base=origin/main --head=HEAD",
     "test": "nx run-many -t test",
+    "test:ci:affected": "nx affected -t test --base=origin/main --head=HEAD",
     "test:browser": "nx run-many -t test:browser",
-    "test:ci:changed": "nx affected -t test --base=origin/main --head=HEAD",
-    "test-all-versions": "nx run-many -t test-all-versions",
+    "test:browser:ci:affected": "nx affected -t test:browser --base=origin/main --head=HEAD",
+    "test-all-versions": "nx run-many --parallel=false -t test-all-versions",
+    "test-all-versions:ci:affected": "nx affected --parallel=false -t test-all-versions --base=origin/main --head=HEAD",
```

Comment:
I have an idea for a tav patch that might make it work in parallel. As I understand, tav operates on the monorepo, so when it does it in parallel, everything breaks.
We could try installing the packages tav needs to install into a package that isn't under "workspaces", and then linking them into the `node_modules` of the specific instrumentation.

Reply:
Trying it out here: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2994
Looks like pino now passes, but some other stuff fails.